meta-oe/recipes-support/spdlog/spdlog_1.5.0.bb

```diff
@@ -3,8 +3,7 @@ HOMEPAGE = "https://github.com/gabime/spdlog/wiki"
 LICENSE = "MIT"
 LIC_FILES_CHKSUM = "file://${COREBASE}/meta/files/common-licenses/MIT;md5=0835ade698e0bcf8506ecda2f7b4f302"
 
-SRCREV = "a7148b718ea2fabb8387cb90aee9bf448da63e65"
-SRC_URI = "git://github.com/gabime/spdlog.git;protocol=git;branch=v1.x;"
+SRC_URI = "git://github.com/gabime/spdlog.git;protocol=git;branch=v1.x;tag=v1.5.0;"
```

Comment:
Let us use SRCREV corresponding to v1.5.0, using tags still forces bitbake to ping the upstream git and breaks  builds where BB_NO_NETWORK is used.

Reply:
Good to know :)